modules/social_features/social_node/social_node.install

```diff
@@ -12,3 +12,24 @@ function social_node_install() {
   // Set the view mode to use when shown in activities.
   activity_creator_set_entity_view_mode('node', 'activity');
 }
+
+/**
+ * Remove "gnode_anonymous" realm for nodes without "public" visibility.
```

Comment:
@nechai Reason of this update is not really clear, could you add more explanation here and in the PR?

Reply:
@ribel After we delete the filter we should also delete this realm for all nodes with "community" or "members" visibility because this realm allows to see these nodes teasers for anonymous (previously the removed filter hide these nodes for anonymous).

Reply:
Ok, clear.